src/index.html

```diff
@@ -11,12 +11,103 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo_image"
+        href="#logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo Logo"
+        />
+      </a>
+
+      <nav class="navigation">
+        <ul class="nav_list">
+          <li class="nav-item">
+            <a
+              href="#apple"
+              class="nav_link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              href="#samsung"
+              class="nav_link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              href="#smartphones"
+              class="nav_link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              data-qa="hover"
+              href="#laptops"
+              class="nav_link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              href="#gadgets"
+              class="nav_link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              href="#tablets"
+              class="nav_link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              href="#photo"
+              class="nav_link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+              href="#video"
+              class="nav_link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
Add empty lines between multiline sibling blocks of HTML to add some 'air' and simplify reading. However, don't add them between parent and child elements.